### ISSUE TITLE
`test_empty_tenant_size` is flaky

### ISSUE THREAD

**Issue by jcsp** [neondatabase/neon#6965]
8  failures in past 48h. This seems quite common, on pg15:

```
AssertionError: tenant_size should not be affected by shutdown of compute (Currently 22208688, expected 22208632)
assert 22208688 == 22208632
```

I suspect this is a remnant of pg15 stabilization effort #2809. Thinking now the assertion has always been flawed: we cannot guarantee that the point in time (related to PITR) and there is zero WAL on the safekeepers still being applied when we read the size. Curiously this is very much more stable on pg14.

The test is marked as XFAIL at least on pg15 for this reason.

It seems to happen with both debug and release builds.

**Comment by koivunej** [comment]
#6712 was merged recently. I wonder if the flakyness on pg15 was the shutdown checkpoint which was not getting through on pg14 and pg16.

**Comment by koivunej** [comment]
Interestingly flakyness has gone down, not up as expected. The single follow-up is still not done.

**Comment by koivunej** [comment]
Fixed with #7294.
